modules/rosa-sharing-vpc-cluster-creation.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * networking/rosa-shared-vpc-config.adoc
+:_content-type: PROCEDURE
+[id="rosa-sharing-vpc-cluster-creation_{context}"]
+= Creating your cluster in a shared VPC
+
+After the AWS account user who initially created the DNS domain and the IAM roles, the VPC-owning AWS account user must create a private hosted zone and update the trust policy on the IAM role that was created for sharing the VPC.
+
+.Procedure
+
+. The AWS account user who owns the VPC must update the VPC sharing IAM role and add the Installer and Ingress Operator roles to the principal section of the trust policy.
++
+[source,terminal]
+----
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+	  "Sid": "Statement1",
+	  "Effect": "Allow",
+	  "Principal": {
+	  	"AWS": [
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/ManagedOpenShift-ingress-operator-cloud-credentials",
+          "arn:aws:iam::<Cluster-Creator's-AWS-Account-ID>:role/ManagedOpenShift-Installer-Role"
+        ]            
+	  },
+	  "Action": "sts:AssumeRole"
+	}
+  ]
+}
+----
+. After updating the trust policy, the VPC-owning AWS account user creates a private hosted zone in the link:https://us-east-1.console.aws.amazon.com/route53/v2/[Route 53 section of the AWS console]. The hosted spaces name is `<cluster-name>.<dns_domain>`. The private hosted zone needs to be associated with the created VPC.
+. After the hosted zone is created, provide the following to the AWS account user who created the cluster:
++
+[source,terminal]
+----
+rosa create cluster --cluster-name <cluster_name> --sts --operator-roles-prefix <prefix> --oidc-config-id <oidc_config_id> --region us-east-1 --subnet-ids <subnet_ids> --private-hosted-zone-id <hosted_zone_ID> --shared-vpc-role-arn <vpc-role-arn> --base-domain <dns-domain>
```

Comment:
I feel like this should be broken into _another_ module, and make it obvious that this is what the *cluster-creator AWS account* user should be running in _their_ terminal.